Description: Add <linux/timer.h> include
 The code checks for the "timer_setup" macro, which is defined in this header.
Author: Simon Richter <sjr@debian.org>
Bug-Debian: https://bugs.debian.org/1108287
Forwarded: no
Last-Update: 2025-07-22

--- v4l2loopback-0.15.0.orig/v4l2loopback.c
+++ v4l2loopback-0.15.0/v4l2loopback.c
@@ -23,6 +23,7 @@
 #include <linux/slab.h>
 #include <linux/fs.h>
 #include <linux/capability.h>
+#include <linux/timer.h>
 #include <linux/eventpoll.h>
 #include <media/v4l2-ioctl.h>
 #include <media/v4l2-common.h>
